examples/ding-dong-bot.kt

```diff
@@ -1,4 +0,0 @@
-/**
```

Comment:
May we keep this file so that a new developer can be easy to find out the main entrance of our Kotlin example, especially a developer who is not very familiar with the Kotlin directory structure?